packages/block-editor/src/components/block-card/style.scss

```diff
@@ -2,6 +2,7 @@
 .block-editor-block-card {
 	display: flex;
 	align-items: flex-start;
+	padding: $grid-size-large;
```

Comment:
This breaks the styling of the card in the inserter.

Reply:
Fixed 👍